app/scripts/metamask-controller.js

```diff
@@ -2168,12 +2168,13 @@ export default class MetamaskController extends EventEmitter {
    * @param {string} snapId - The ID of the snap the handler is being triggered on.
    * @param {string} handler - The handler to trigger on the snap for the request.
    * @param {boolean} success - Whether the invocation was successful or not.
+   * @param {string} origin - The origin of the request.
    */
   _trackSnapExportUsage = wrap(
     memoize(
       () =>
         throttle(
-          (snapId, handler, success) =>
+          (snapId, handler, success, origin) =>
```

Comment:
Wondering if we should throttle for every snap + handler combination or snap + handler + origin combination?

Your current change would still throttle by snap + handler. WDYT?

Reply:
Yeah, it makes sense to throttle with snap + handler + origin.